editor/themes/editor_theme_manager.cpp

```diff
@@ -1161,6 +1161,8 @@ void EditorThemeManager::_populate_standard_styles(const Ref<EditorTheme> &p_the
 		p_theme->set_stylebox("button_pressed", "TabBar", p_config.panel_container_style);
 		p_theme->set_stylebox("button_highlight", "TabBar", p_config.panel_container_style);
 
+		Color icon_color = p_config.dark_theme ? Color(1, 1, 1) : Color(0, 0, 0);
```

Comment:
`Color(0, 0, 0)` will make icons solid black, without any detail, probably not the best default.

<img width="218" height="76" alt="Image" src="https://github.com/user-attachments/assets/3c46d11f-823d-4db8-a3b9-ee017ca74627" />